packages/cli/src/lib/yargs-cli.integration.test.ts

```diff
@@ -1,69 +1,104 @@
-import { dirname, join } from 'path';
-import { fileURLToPath } from 'url';
 import { describe, expect, it } from 'vitest';
-import { objectToCliArgs } from '@code-pushup/utils';
-import { middlewares } from './middlewares';
+import { CoreConfig } from '@code-pushup/models';
+import {
+  CoreConfigCliOptions,
+  GeneralCliOptions,
+} from './implementation/model';
 import { options } from './options';
 import { yargsCli } from './yargs-cli';
 
 describe('yargsCli', () => {
-  it('global options should provide correct defaults', async () => {
-    const args: string[] = [];
-    const parsedArgv = await yargsCli(args, {
+  it('should provide correct default values for global options', async () => {
+    const parsedArgv = await yargsCli<GeneralCliOptions>([], {
       options,
     }).parseAsync();
     expect(parsedArgv.verbose).toBe(false);
     expect(parsedArgv.progress).toBe(true);
   });
 
-  it('multiple config args should be parsed to last item from array', async () => {
-    const args: string[] = ['--config=./config.a.ts', '--config=./config.b.ts'];
-    const parsedArgv = await yargsCli(args, {
+  it('should parse a single boolean negated argument', async () => {
+    const parsedArgv = await yargsCli<GeneralCliOptions>(['--no-progress'], {
       options,
     }).parseAsync();
-    expect(parsedArgv.config).toBe('./config.b.ts');
+    expect(parsedArgv.progress).toBe(false);
   });
 
-  it('single config arg should be parsed as a single string', async () => {
-    const args: string[] = ['--config=./config.a.ts'];
-    const parsedArgv = await yargsCli(args, {
-      options,
-    }).parseAsync();
+  it('should parse a single config argument as a string', async () => {
+    const parsedArgv = await yargsCli<GeneralCliOptions>(
+      ['--config=./config.a.ts'],
+      {
+        options,
+      },
```

Comment:
Tiny formatting improvement, saves some real estate.

The trick is to put the 1st property on the same line as `{` before you format, then Prettier will try to put it on one line. But if the 1st property is already on another line, Prettier always formats it as multiline.

```suggestion
      { options },
```

Reply:
I didn't know about this, that's a game changer 😮